INSTALL

```diff
@@ -56,6 +68,11 @@ How to install PBS Pro using the configure script.
   following command:
 
     ./configure --prefix=/opt/pbs --libexecdir=/opt/pbs/libexec
+    
+  For Debian systems you should run the following command:
+
+    CFLAGS='-I/usr/include/tcl8.6' \
+      LDFLAGS='-L/usr/lib/x86_64-linux-gnu -ltclstub8.6' ./configure
```

Comment:
CFLAGS and LDFLAGS  are no more needed. I removed tcl.h from AC_CHECK_HEADERS in configure.ac, which always failed. Correct paths added to m4/with_tcl.m4.